docs/Future_vision.md

```diff
@@ -0,0 +1,43 @@
+# Future Vision
+
+## Where to go from here?
+
+Liquid still hasn't reached its full potential. We wanted to do a lot more than we managed to do in this semester. You can look through our user stories on github or zenhub (please beware that the user stories project on github is outdated, because we switched to zenhub). To present a clear overview here is a short list (please refer to the [contributing doc](https://github.com/hpi-swa-teaching/Liquid/blob/main/CONTRIBUTING.md) for term definitions):
+
+-   ### Multiple questions per poll
+    Initially we wanted to implement multiple questions per poll. Our client did not deem this necessary for the time being. But because we already started making the proper infrastructure for this function, we decided to leave it there for possible future developers. For example, our poll-draft object still has a 'questionList'.
```

Comment:
```suggestion
Our initial design included polls with multiple questions, which the customer didn't deem neccessary. Data structures to support multiple questions were left in place, as an exercise the reader (that's **you**!). For example, our poll-draft object still has a 'questionList', instead of a `question`.
```